src/ui/main-window/ThumbnailView.swift

```diff
@@ -17,7 +17,7 @@ class ThumbnailView: NSStackView {
     var quitIcon = TrafficLightButton(.quit, NSLocalizedString("Quit app", comment: ""), windowsControlSize)
     var closeIcon = TrafficLightButton(.close, NSLocalizedString("Close window", comment: ""), windowsControlSize)
     var minimizeIcon = TrafficLightButton(.miniaturize, NSLocalizedString("Minimize/Deminimize window", comment: ""), windowsControlSize)
-    var maximizeIcon = TrafficLightButton(.fullscreen, NSLocalizedString("Fullscreen window", comment: ""), windowsControlSize)
+    var maximizeIcon = TrafficLightButton(.fullscreen, NSLocalizedString("Fullscreen/Defullscreen window", comment: ""), windowsControlSize)
```

Comment:
I think this change is good. However, I would like to point out that this will be quite impactful on the localizations.  The app is translated in 50 languages, and this little change means that at least 50 people will have to come in and provide a translation.

It's probably still right to make this change. I just wanted to share the full picture of the consequences with you

Reply:
Actually, this string already exists in `ControlsTab.swift`. So you fixed the usage without any impact on the localization side 👍